docs/source/api/deliveryservice_server_capabilities.rst

```diff
@@ -0,0 +1,222 @@
+	| sortOrder         | no       | Changes the order of sorting. Either ascending (default or "asc") or descending ("desc")                      |
+	+-------------------+----------+---------------------------------------------------------------------------------------------------------------+
+	| limit             | no       | Choose the maximum number of results to return                                                                |
+	+-------------------+----------+---------------------------------------------------------------------------------------------------------------+
+	| offset            | no       | The number of results to skip before beginning to return results. Must use in conjunction with limit.         |
+	+-------------------+----------+---------------------------------------------------------------------------------------------------------------+
+	| page              | no       | Return the n\ :sup:`th` page of results, where "n" is the value of this parameter, pages are ``limit`` long   |
+	|                   |          | and the first page is 1. If ``offset`` was defined, this query parameter has no effect. ``limit`` must be     |
+	|                   |          | defined to make use of ``page``.                                                                              |
+	+-------------------+----------+---------------------------------------------------------------------------------------------------------------+
+
+.. code-block:: http
+	:caption: Request Example
+
+	GET /api/1.4/deliveryservice_server_capabilities HTTP/1.1
+	Host: trafficops.infra.ciab.test
+	User-Agent: curl/7.47.0
+	Accept: */*
+	Cookie: mojolicious=...
+
+Response Structure
+------------------
+:deliveryServiceID: The :term:`Delivery Service`'s ID
+:xmlID:             The :term:`Delivery Service`'s :ref:`ds-xmlid`
+:lastUpdated:       The date and time at which this association between the :term:`Delivery Service` and the Server Capability was last updated, in an ISO-like format
+:serverCapability:  The Server Capability's name
+
+.. code-block:: http
+	:caption: Response Example
+
+	HTTP/1.1 200 OK
+	Access-Control-Allow-Credentials: true
+	Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept, Set-Cookie, Cookie
+	Access-Control-Allow-Methods: POST,GET,OPTIONS,DELETE
+	Access-Control-Allow-Origin: *
+	Content-Type: application/json
+	Set-Cookie: mojolicious=...; Path=/; HttpOnly
+	Whole-Content-Sha512: UFO3/jcBFmFZM7CsrsIwTfPc5v8gUiXqJm6BNp1boPb4EQBnWNXZh/DbBwhMAOJoeqDImoDlrLnrVjQGO4AooA==
+	X-Server-Name: traffic_ops_golang/
+	Date: Mon, 07 Oct 2019 22:15:11 GMT
+	Content-Length: 396
+
+	{
+		"response": [
+			{
+				"deliveryServiceID": 1,
+				"lastUpdated": "2019-10-07 22:05:31+00",
+				"serverCapability": "ram"
+				"xmlId": "example_ds-1",
+			},
+			{
+				"deliveryServiceID": 2,
+				"lastUpdated": "2019-10-07 22:05:31+00",
+				"serverCapability": "disk"
+				"xmlId": "example_ds-2,
```

Comment:
Ah, this is the problem. It's not the HTTP part that's wrong, it's this incorrect JSON that got pushed onto the context stack. That comma is on the wrong line.